### ISSUE TITLE
observable:firstName needs to contextualize as pertaining to contact, not identity:Person

### ISSUE THREAD

**Issue by ajnelson-nist** [ucoProject/UCO#350]
## Background

`observable:firstName` currently has the following definition, added in CP-5 (refactoring contacts):

> The first name of a person.

It used to have the following definition:

> The first name of the contact.

There was no specific rationale given for `observable:firstName` in the [proposal](https://unifiedcyberontology.atlassian.net/wiki/spaces/OC/pages/329089025/UCO+CP-5+Refactor+and+improve+contacts).  (This proposal has not been publicly exported, as became practice in a later UCO release.)  This is the only focused remark on `observable:firstName`:

> Modify definition comment of observable:firstName to abstract it from only Contacts

Someone reached out to me to ask when `observable:firstName` would be used instead of something from the `identity` namespace.  I don't have a good answer for them.

## Requested resolution

Can an example be created, or referenced, that relates some `observable:ObservableObject` subclass, which uses `observable:firstName`, to an `identity:Person`?

AND/OR

Can the definition of `observable:firstName` revert to `The first name of the contact.`?

AND/OR

Can the name of `observable:firstName` change to `observable:contactFirstName`, to separate scope of this property clearly away from `identity:Person` and more towards a contact *record*?

**Comment by ajnelson-nist** [comment]
I did a `git-blame` dive and found this was the history of the line, back to 2017 when the Turtle files were first made (newest commit to oldest):

| Commit | Definition effected by commit |
| --- | --- |
| 1633fd07 | The first name of a person. |
| a33b8fe5 | The first name of the contact. |
| 81e1a6b0 | The first name of a person. |
| bc4cf8e4 | The first name of the contact. |
| ff756719 | The first name of the contact. |
| 68812a02 | The first name of the contact. |
| 25338b3c | The first name of the contact. |
| d5e6790c | The first name of the contact. |

**Comment by sbarnum** [comment]
I disagree that there is any ambiguity here but would not object to the proposed changes to firstName, lastName, middleName, namePhonetic, namePrefix, nameSuffix or nickname. It constrains the flexibility for these properties to be used for any other observable objects in the future but we can change them back in the future if needed or just create different versions for each object which we have tried to avoid..

observable:displayName should definitely not be changed. It is used in 8 other classes and is named and defined appropriately.

**Comment by ajnelson-nist** [comment]
> observable:displayName should definitely not be changed. It is used in 8 other classes and is named and defined appropriately.

I'm fine removing this from the proposal.  I'll do so now.  Thanks for the context.

**Comment by ajnelson-nist** [comment]
From discussion in the OC meeting today, this proposal should be updated to include a review of on what `Facet`s these properties all appear on.  `displayName` doesn't seem to have been the only confusion.

**Comment by ajnelson-nist** [comment]
This proposal raises an opportunity to review the `accounts.json` example on CASE-Examples.

**Comment by ajnelson-nist** [comment]
This slipped through my review queue.  Due to timing, it is now part of the UCO 2.0.0 milestone.
